ReadMe.txt

```diff
@@ -301,7 +301,7 @@ Commands:
   mw <addr>             - Memory watch at addr
   ms <value>            - Search value in RAM memory
   mr <value>            - Refine search with new value
-  mp                    - Print memory search addresses
+  p                     - Print memory search addresses
```

Comment:
Here I vote for 'mp' instead of single 'p'. This way it will be easier to remember.

Reply:
I agree, but I couldn't make it work with command repetition :(